client/src/components/Search/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { InputPrefix, InputGroup, TextInput } from '@trussworks/react-uswds';
+
+import SearchSvg from '../../assets/sprite.svg';
+
+interface InputPrefixProps {
+  placeholder?: string;
+}
+
+export function Search({ placeholder = 'Search' }: InputPrefixProps) {
+  return (
+    <InputGroup className="mx-0">
```

Comment:
It seems like we're good without this `classname`.
```suggestion
    <InputGroup>
```